source/nordic-sdk/components/softdevice/s130/headers/nrf_svc.h

```diff
@@ -49,7 +49,7 @@
   { \
     __asm( \
         "svc %0\n" \
-        "bx r14" : : "I" ((uint32_t)number) : "r0" \
```

Comment:
@andresag01 are these warnings new? I don't see any change in this PR to generate new warnings.